pkg/asset/asset.go

```diff
@@ -36,9 +36,11 @@ const (
 // AssetConfig holds all configuration needed when generating
 // the default set of assets.
 type Config struct {
-	EtcdServers []*url.URL
-	APIServers  []*url.URL
-	AltNames    *tlsutil.AltNames
+	EtcdServers    []*url.URL
+	APIServers     []*url.URL
+	CAFile         string
+	PrivateKeyFile string
```

Comment:
I think we're providing the contents (or we could provide as types), not the path.